### DISCUSSION TITLE
Using geckos.io with javascript?

### DISCUSSION THREAD

**Discussion by julifos** [geckosio/geckos.io#179] [Help / Questions] [answered]
Sorry if this is a silly question, but I don't know much about typescript: can be geckos.io used with javascript instead of typescript?

**Comment by yandeu** [comment, marked as answer]
Yes. Geckos.io is written in TS, but exported as JS.

**Reply by julifos** [reply]
Thanks!
In fact I'm not sure what I'm asking ;-) Maybe this question would suit better in a Node forum?
I know the part of the client is pure javascript. I guess if I can use javascript in the server-side, instead of typescript. If I understand correctly, typescript is compiled (= converted to javascript) and executed afterwards. Is there some easy way to circumvent that step? It's only that I'm a bit reluctant to learn new stuff if I don't need to.
